lib/services/database.dart

```diff
@@ -55,4 +58,36 @@ class DatabaseService {
   Stream<DocumentSnapshot> get userData {
     return userDetails.document(uid).snapshots();
   }
+
+  // add group details
+  Future<void> createTrip(RequestDetails requestDetails) async {
+    var user = await _auth.currentUser();
+    final docRef = await requestdetails.add({
+      'destination': requestDetails.destination.toString(),
+      'startDate': requestDetails.startDate.toString(),
+      'startTime': requestDetails.startTime.toString(),
+      'endDate': requestDetails.endDate.toString(),
+      'endTime': requestDetails.endTime.toString(),
```

Comment:
Dont use toString. Use timestamp instead. See how to directory push DateTime object to the firebase

Reply:
Need to figure this out. Will try in next PR. Please review the rest.